maat-court-data-api/src/main/java/gov/uk/courtdata/controller/ReservationsController.java

```diff
@@ -43,4 +51,39 @@ public ResponseEntity<Boolean> isMaatRecordLocked(
         log.info(String.format("Check if maatId is locked - %d {}", maatId));
         return ResponseEntity.ok(reservationsRepositoryHelper.isMaatRecordLocked(maatId));
     }
+
+
+    @GetMapping(value = "/recordname/{recordName}/recordid/{recordId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Get the reservation status object of a record")
```

Comment:
Description could be more accurate, looks like it was copy pasted from previous endpoint.